src/seedu/addressbook/data/tagging/Tagging.java

```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.data.tagging;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
```

Comment:
This class is not called by any other classes. According to the diagram, this class should be instantiated and kept track of by AddressBook.